elastica/boundary_conditions.py

```diff
@@ -317,6 +328,206 @@ def __init__(self, *fixed_data, **kwargs):
             # transpose from (blocksize, dim, dim) to (dim, dim, blocksize)
             self.fixed_directors = np.array(dir).transpose((1, 2, 0))
 
+        translational_constraint_selector = kwargs.get(
```

Comment:
Don't query `kwargs` directly. It is error-prone in the downstream code.

Set `translational_constraint_selector` as a parameter of `__init__`, so that any other class in MRO doesn't see the parameter. It is how `fixed_positions` and `fixed_directors` are written.

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/143/commits/03fc1502110db67904ceb26648b378d982e6cdf1